### PR TITLE
Update configuring-the-referrer-policy-for-your-enterprise.md

### PR DESCRIPTION
**Pull request by sgarciajaramillo**
### Why:

There is a step missing to find where the USer Agent Referrer Policy setting is loacted.

Closes: 
N/A

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adding additional step to get into the Authentication security section where this setting is located

**Setting location on GHES 3.10.3**
image

**Before**
image

**After**
image

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by prechayimmee — COMMENTED** [review]
[maintainer edited to remove spammy content]

**Review by rolandtmz814 — COMMENTED** [review]
Thanks

File: content/admin/configuration/hardening-security-for-your-enterprise/configuring-the-referrer-policy-for-your-enterprise.md [hunk 1]
```diff
@@ -34,5 +34,6 @@ You can enable the `same-origin` referrer policy to instruct modern browsers to
 
 {% data reusables.enterprise-accounts.access-enterprise %}
 {% data reusables.enterprise-accounts.settings-tab %}
```
Comment (rolandtmz814) [inline content/admin/configuration/hardening-security-for-your-enterprise/configuring-the-referrer-policy-for-your-enterprise.md:36]: Will do

File: content/admin/configuration/hardening-security-for-your-enterprise/configuring-the-referrer-policy-for-your-enterprise.md [hunk 2]
```diff
@@ -34,5 +34,6 @@ You can enable the `same-origin` referrer policy to instruct modern browsers to
...
+1. Cick on "Authentication security"
```
Comment (mattpollard) [inline content/admin/configuration/hardening-security-for-your-enterprise/configuring-the-referrer-policy-for-your-enterprise.md:37]:
```suggestion
1. Under {% octicon "gear" aria-hidden="true" %} **Settings**, click **Authentication security**.
```

**Review by mattpollard — DISMISSED** [review]
Thanks, @sgarciajaramillo ⚡ I've verified that this is correct for GHES 3.7+. I'll incorporate a small revision, and then publish the change.

**Review by mattpollard — APPROVED** [review]

**Pull request merged**